modules/hal_silabs/wiseconnect/CMakeLists.txt

```diff
@@ -62,5 +62,48 @@ zephyr_library_sources_ifdef(CONFIG_DMA_SILABS_SIWX91X
   ${WISECONNECT_DIR}/components/device/silabs/si91x/mcu/drivers/rom_driver/src/rsi_rom_table_si91x.c
 )
 
+if(CONFIG_WISECONNECT_NETWORK_STACK)
+
```

Comment:
remove newline, indent with 2 spaces

Reply:
I have reformatted all the `CMakeLists.txt` of this PR.